### PR TITLE
Support Split FFs and Freemium in CWS Spinwicks

### PR DESCRIPTION
**Pull request by nickmisasi**
#### Summary
This PR does 2 things:

1. Adds support for Freemium, through the `CLOUD_DEFAULT_PRODUCT_ID` and `CLOUD_DEFAULT_TRIAL_PRODUCT_ID` env variables.
2. Adds support for Split.io out of the box. Spinwicks will be created with a split server ID equivalent to namespace name (I've added this to the output comment for easy reference)

#### Ticket Link
N/a
#### Release Note

```release-note
None
```

### CODE REVIEW

File: server/spinwick.go
Comment (JulienTant) [inline server/spinwick.go:289]: nit: we could reuse SplitIO wording for this.

```suggestion
	msg := fmt.Sprintf("CWS test server created! :tada:\n\nAccess here: %s\n\nSplit individual target: %s", spinwickURL, deployment.Environment.CWSSplitServerID)
```

**Review by JulienTant — APPROVED** [review]
LGTM @nickmisasi! Super convenient to have SplitIO working out of the box!

**Review by marianunez — APPROVED** [review]

**Comment by JulienTant** [comment]
gentle ping @stylianosrigas

**Review by stylianosrigas — APPROVED** [review]
LGTM, once merged please let me know to move forward with the release :)

**Pull request merged**
